### DISCUSSION TITLE
copilot gives the code in comments

### DISCUSSION THREAD

**Discussion by hilalmr** [community/community#11116] [Copilot Conversations]
Hi I am a junior game developer and a programmer,

I have faced the problem where the copilot gives the code in comments

### PICTURE: 

![image](https://user-images.githubusercontent.com/72068174/152730903-4c6df65d-02f4-442f-bb85-cb8824ee2e3b.png)

```
#make a music

#import pygame.mixer
#import pygame.mixer_music
#import pygame.mixer
#import pygame.mixer_music
#import pygame.mixer
```

only the `#make a music` is typed by me
rest are done by the copilot

**Comment by KTibow** [comment]
This might be due to the fact you put no space between the `#` and the `make a music`; usually, people add a space there. If Copilot is still stuck, try "nudging" it by removing the comment symbol. It'll take the context, and provide you with normal code.

**Reply by hilalmr** [reply]
Thanks for your assistance; it is now working.
